child_compassion/wizards/child_search_attributes.py

```diff
@@ -0,0 +1,124 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+from openerp import models, api, fields, _
+
+
+class ImportReview(models.TransientModel):
+
+    _name = 'child.search.attributes'
```

Comment:
This name is not very good... what about `child.hold.wizard`? Because these are not search attributes, but rather hold parameters. If you find another name you can as well propose it. Rename as well the references to the object in the views and the code.